src/libraries/System.Runtime/tests/System.Runtime.Tests/System/TypeLoadExceptionTests.cs

```diff
@@ -36,5 +37,28 @@ public static void Ctor_String_Exception()
             ExceptionHelpers.ValidateExceptionProperties(exception, hResult: COR_E_TYPELOAD, innerException: innerException, message: message);
             Assert.Equal("", exception.TypeName);
         }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotMonoRuntime))]
+        public static void TypeLoadExceptionMessageContainsMethodNameWhenInternalCallOnlyMethodIsCalled()
+        {
+            AssertExtensions.ThrowsContains<TypeLoadException>(() => new F1(), "Internal call method 'F2.Foo' with non_NULL RVA.");
```

Comment:
This test would fit better under src\tests. src\tests is for runtime tests. src\libraries\*\tests is for API tests. This is not an API test.